caluma/caluma_form/tests/test_filter_by_answer.py

```diff
@@ -157,6 +157,43 @@ def test_query_all_questions(
     )
 
 
+@pytest.mark.parametrize(
+    "question__type,values,search,expect_count",
+    [
+        (models.Question.TYPE_TEXT, ["foo", "bar", "baz"], ["foo", "foobar"], 1),
+        (models.Question.TYPE_TEXT, ["foo", "bar", "baz"], ["foo", "baz"], 2),
+        (models.Question.TYPE_TEXT, ["foo", "bar", "baz"], [], 0),
+        (models.Question.TYPE_INTEGER, [1, 10, 100], [1, 100], 2),
+    ],
+)
+def test_has_answer_in(
+    schema_executor, db, question, answer_factory, values, search, expect_count
+):
+
+    for value in values:
+        answer_factory(question=question, value=value)
+
+    query = """
+        query asdf ($hasAnswer: [HasAnswerFilterType]!) {
+          allDocuments(hasAnswer: $hasAnswer) {
```

Comment:
Please use new-style filter syntax - this one will be removed soon.
```suggestion
          allDocuments(filter: [{hasAnswer: $hasAnswer}]) {
```